lesson05/circle.go

```diff
@@ -0,0 +1,31 @@
+package main
```

Comment:
It looks like you based this PR on you lesson05 branch. You should create you PR based on your main branch, not other PR branch.